src/callbacks/OTP.ts

```diff
@@ -0,0 +1,11 @@
+import { OTP, PostOTP } from "../api/Auth";
+
+export const OTPCallback = async (input: OTP) : Promise<boolean> => {
+	const response = await PostOTP(input);
+	if (response.Status == 200) {
+		return true;
+	}else {
+		console.error(response.Payload);
```

Comment:
Have consistent error handling. `LoginCallback` and `OTPCallback`/`SignupCallback` are handling differently.

Reply:
Check everywhere where you have used `console.error`